packages/app/src/components/scaffolder/customScaffolderExtensions.tsx

```diff
@@ -68,7 +68,7 @@ export const LowerCaseValuePickerFieldExtension = scaffolderPlugin.provide(
 const MockDelayComponent = (
   props: NextFieldExtensionComponentProps<{ test?: string }>,
 ) => {
-  const { onChange, formData, rawErrors } = props;
+  const { onChange, formData, rawErrors = [] } = props;
```

Comment:
`rawErrors` can now be undefined, meaning `rawErrors?.length > 0` no longer makes sense if `rawErrors` can be undefined.